src/components/constraints/horizontal/horizontal.tsx

```diff
@@ -18,21 +22,21 @@ const getConstraintSyle = constraint => {
     case 'scale':
       return styles.constraintScale;
     default:
-      return undefined;
+      throw new Error(`Unsupported constraint ${constraint}`);
   }
 };
 
-const Horizontal = ({ children, constraint }) => (
-  <div className={classnames(styles.container, getConstraintSyle(constraint))}>
-    {children}
+const Horizontal: React.SFC<HorizontalProps> = props => (
```

Comment:
I think we should use SF. SFC is deprecated in TS.

Reply:
Sure, thanks for spotting that!

PS: this was generated using the plugin https://github.com/lyft/react-javascript-to-typescript-transform, which I guess is a bit outdated.